### PR TITLE
Random updates

### PR DESCRIPTION
**Pull request by SoTMaulder**
Added several maps, latest offsets, added friends (manually put 2 in for example), switched filter to guns/ammo. (can switch manually) added black background for fuser users

### CODE REVIEW

**Review by Enoz — CHANGES_REQUESTED** [review]
Thanks for the changes! Some

File: dma-dayz-cpp/DayZ/DayZMem/RadarAdapter/RadarAdapter.cpp [hunk 1]
```diff
@@ -13,70 +25,135 @@ void DayZ::RadarAdapter::drawLoot(DMARender::IGameMap* curMap, const DMARender::
...
+void DayZ::RadarAdapter::drawAliveEntities(DayZ::Camera* camera, DMARender::IGameMap* curMap, const DMARender::MapTransform& mTransform, const std::vector<std::shared_ptr<DayZ::Entity>>& entities, Scoreboard* scoreboard)
 {
+	//std::ifstream myfile ("config.txt");
+	//std::string mystring;
+	///std::cout << "test" << mystring << std::endl;
+
+
+	std::vector<std::string> MainPlayerPlayingNames = { "LOCALPLAYERNAME1", "LOCALPLAYERNAME2" };
+
+	std::vector<std::string> specialPlayerNames = { "Bkmmhz", "BuckHammond", "Friend3", "Friend4", "Friend5", "Friend6" };
```
Comment (Enoz) [inline dma-dayz-cpp/DayZ/DayZMem/RadarAdapter/RadarAdapter.cpp:85]:
```suggestion
	std::vector<std::string> specialPlayerNames = { "Friend1", "Friend2", "Friend3", "Friend4", "Friend5", "Friend6" };
```

[the remark this answers is not in the document]
Comment (SoTMaulder) [reply dma-dayz-cpp/DayZ/DayZMem/RadarAdapter/RadarAdapter.cpp:85]: Lol these were 2 random players i found in my game next to my spawn just to test

File: dma-dayz-cpp/DayZ/DayZMem/RadarAdapter/RadarAdapter.cpp [hunk 2]
```diff
@@ -13,70 +25,135 @@ void DayZ::RadarAdapter::drawLoot(DMARender::IGameMap* curMap, const DMARender::
...
+				//specialItemGuns = std::find(Listofgunnames.begin(), Listofgunnames.end(), bestName2->value) != Listofgunnames.end();
+				//specialItemBuilding = std::find(ListofBuildingItems.begin(), ListofBuildingItems.end(), bestName2->value) != ListofBuildingItems.end();
+				//specialItemIgnore = std::find(ListofIgnoreItems.begin(), ListofIgnoreItems.end(), bestName2->value) != ListofIgnoreItems.end();
+				//specialItemAmmo = std::find(Listofammonames.begin(), Listofammonames.end(), bestName2->value) != Listofammonames.end();
+			}
+		}*/
+		
 		if (item->isPlayer()) {
-			textCol = IM_COL32(0, 255, 255, 255);
+			textCol = IM_COL32(0, 255, 255, 100);
```
[the remark this answers is not in the document]
Comment (SoTMaulder) [reply dma-dayz-cpp/DayZ/DayZMem/RadarAdapter/RadarAdapter.cpp:51]: kk i switched it back.

**Review by Enoz — APPROVED** [review]

**Pull request merged**